### DISCUSSION TITLE
Background color of element slightly different from listview color

### DISCUSSION THREAD

**Discussion by lars-vc** [davatorium/rofi#1651] [Q&A] [answered]
I want to make the listview of rofi seethrough and am therefor using picom to get transparency. But when I set a background color, the color of the elements seem slightly different from the listview color.
Secondly only alpha values going from 00 to 0F seem to do something, but going higher seems to turn off transparency. And 0F is practically completely seethrough
Here is the config I am using. While writing this I noticed I can't launch a screenshot app and keep rofi open (kbd shortcuts don't work and clicking the icon in the tray closes rofi) so I don't know how to illustrate it right now.
If anyone could help with these small issues that would be greatly appreciated!
```
configuration {
  display-drun: "Applications:";
  display-window: "Windows:";
  drun-display-format: "{icon} {name}";
  font: "JetBrainsMono Nerd Font Medium 10";
  modi: "window,run,drun";
  show-icons: true;

  kb-row-up: "Up,Control+k,Shift+Tab,Shift+ISO_Left_Tab";
  kb-row-down: "Down,Control+j";
  kb-accept-entry: "Control+m,Return,KP_Enter";
  terminal: "mate-terminal";
  kb-remove-to-eol: "Control+Shift+e";
  kb-mode-complete: "Control+Shift+l";
  kb-mode-next: "Shift+Right,Control+Tab,Control+l";
  kb-mode-previous: "Shift+Left,Control+Shift+Tab,Control+h";
  kb-remove-char-back: "BackSpace";
}

@theme "/dev/null"

* {
  bg: #11121D0F;
  bg-alt: #444b6a;

  fg: #FFFFFF;
  fg-alt: #787c99;

  background-color: @bg;
  
  border: 0;
  margin: 0;
  padding: 0;
  spacing: 0;
}

window {
  width: 40%;
}

element {
  padding: 8 0;
  text-color: @fg-alt;
}

element selected {

  background-color: @bg;
  text-color: @fg;
}

element-text {
  background-color: inherit;
  text-color: inherit;
  vertical-align: 0.5;
}

element-icon {
  size: 30;
}

entry {
  background-color: @bg-alt;
  padding: 12;
  text-color: @fg;
}

inputbar {
  children: [prompt, entry];
}

listview {
  padding: 8 12;
  background-color: @bg;
  columns: 1;
  lines: 8;
}

mainbox {
  background-color: @bg;
  children: [inputbar, listview];
}

prompt {
  background-color: @bg-alt;
  enabled: true;
  padding: 12 0 0 12;
  text-color: @fg;
}

/* vim: ft=sass

```

**Comment by DaveDavenport** [comment, marked as answer]
rofi has a build in screenshot function.  Alt+Shift+s by default.

Rofi draws layered, so if you have multiple partial transparent widgets above eachother you will see this. 
See: https://github.com/davatorium/rofi/blob/next/doc/Notes/Theme/Transparency/theme3-transparency.markdown

**Reply by lars-vc** [reply]
Thanks a lot!
